src/untyped.ts

```diff
@@ -18,8 +20,12 @@ module.exports = {
     'no-unbound-method': false,
     // Warns when a namespace qualifier (A.x) is
     'no-unnecessary-qualifier': false,
+    // Warns if a type assertion does not change the type of an expression.
```

Comment:
It'd be useful for the missing rules script to flag rules that require type info (not v1, of course).